common/source/docs/common-pozyx.rst

```diff
@@ -60,15 +60,17 @@ Connect with a Ground Station and set the following parameters:
 - set :ref:`EK3_SRC1_POSXY <EK3_SRC1_POSXY>` to 4 (Beacon)
 - set :ref:`EK3_SRC1_VELXY <EK3_SRC1_VELXY>` to 0 (None)
 - set :ref:`EK3_SRC1_VELZ <EK3_SRC1_VELZ>` to 0 (None)
+- set :ref:`EK3_SRC1_POSZ<EK3_SRC1_POSZ>` to  4 (Beacon)
+- set :ref:`EK3_SRC1_YAW<EK3_SRC1_YAW>` to 6 (ExternalNav)
```

Comment:
EK3_SRC1_YAW should be 1 (Compass) because sadly these beacon systems (pozyx, nooploop, marvelmind) can't provide yaw.

Otherwise the improvements on this page look great.  Nice to see the replacement of "pixhawk" and consolidation on "serial" instead of telemetry.